cpu/native/include/candev_linux.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2016 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     native_cpu
+ * @ingroup     drivers_can
+ * @defgroup    candev_linux SocketCAN driver
+ * @{
+ *
+ * @file
+ * @brief       Implementation of simulated CAN controller driver using SocketCAN on Linux
+ *
+ * @author      Hermann Lelong <hermann@otakeys.com>
+ * @author      Aurelien Gonce <aurelien.gonce@altran.com>
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @}
+ */
+
+#ifndef CANDEV_LINUX_H
+#define CANDEV_LINUX_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if defined(__linux__) /* SocketCAN is supported only on Linux */ || defined(DOXYGEN)
+
+#include <stdbool.h>
+
+#include "can/candev.h"
+#include "mutex.h"
+
+/**
+ * Maximum size of an interface name
+ */
+#define CAN_MAX_SIZE_INTERFACE_NAME (5)
```

Comment:
This is the string length for storing the interface name, not the number of interfaces.